templates/recording-deployment.yaml

```diff
@@ -77,13 +77,15 @@ spec:
         volumeMounts:
         - mountPath: "/data"
           name: data-recordings
+        - mountPath: "/tmp"
+          name: tmp
         resources:
           limits:
-            cpu: {{ .Values.recording.cpu | default "256m"}}
-            memory: {{ .Values.recording.memory | default "256Mi"}}
+            cpu: {{ .Values.recording.cpu | default "1000m"}}
+            memory: {{ .Values.recording.memory | default "1G"}}
           requests:
-            cpu: {{ .Values.recording.cpu | default "256m"}}
-            memory: {{ .Values.recording.memory | default "256Mi"}}
+            cpu: {{ .Values.recording.cpu | default "1000m"}}
+            memory: {{ .Values.recording.memory | default "1G"}}
```

Comment:
1GB of RAM, 1 vCPU. I haven't tested any other configurations -- I just know that 256m was not enough without chromium crashing more than half the time.